app/src/main/java/com/keylesspalace/tusky/service/SendTootService.kt

```diff
@@ -244,8 +251,8 @@ class SendTootService : Service(), Injectable {
     }
 
     private fun saveTootToDrafts(toot: TootToSend) {
-
-        draftHelper.saveDraft(
+        serviceScope.launch {
```

Comment:
Maybe easier to just block? I think that's how services are supposed to work anyways.

Reply:
No they run on the main thread 😬

Reply:
Ah, true dat, we don't have separate process